#
# MARIS Vision - ROS Packages of Vision System used in MARIS project. 
# Copyright (C) 2016 Fabjan Kallasi and Dario Lodi Rizzini.
#
# This file is part of MARIS Vision. 
# Visit http://rimlab.ce.unipr.it/Maris.php for more information about 
# the project. 
#
# MARIS Vision is free software: you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
# 
# MARIS Vision is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with MARIS Vision.  If not, see <http://www.gnu.org/licenses/>.
#
# This represents a 3D line with a 3D Point and a 3D Direction
Header header
geometry_msgs/Point p0
geometry_msgs/Point d
